lib/ops/opsservice/report.go

```diff
@@ -41,7 +41,21 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func (s *site) getSiteOperationCrashReport(op ops.SiteOperation) (io.ReadCloser, error) {
+func (s *site) getClusterReport() (io.ReadCloser, error) {
+	op, err := storage.GetLastOperationForCluster(s.backend(), s.domainName)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	switch op.Type {
+	case ops.OperationInstall:
+		return s.getClusterInstallReport((ops.SiteOperation)(*op))
```

Comment:
This condition lacks the test that the operation is still active - it does not make sense to go that route for a completed operation. This is TBD.

Reply:
Hm, this means that if the cluster has been installed and hasn't undergone any other operations yet, `gravity report` will generate install-specific report? Will you add this check for an active operation before merging?

Reply:
Yes, I will.